feature: add docs to composite credential (#111)
UnumID/UnumID.github.io: docs/acceptance-guide.mdx

Once you join, our team will identify opportunities to promote your brand within our network for 1-click conversions. These can be in-app offers, marketing emails, or similar promotions to drive users to your brand from complementary services.

In brief: As an acceptor in the Verified Inc. network, you will…

- Dramatically improve onboarding conversion.
- Delight users with a 1-click setup.
- Join a network of companies to help you grow faster.

## UX Strategies

> The check to see if the user has credentials that you can accept can happen fully in the background by calling [/hasMatchingCredentials](/api-overview#check-user-credentials) at an opportune time. If the response is contains `match`: `true` then you will redirect to the Verified Inc. Wallet. The demo below demonstrates an example UX.

Reviewer: ```suggestion
You can check if a user has credentials in the background by calling [/hasMatchingCredentials](/api-overview#check-user-credentials) at any time. If the response contains `match`: `true` then you should redirect the user to the `url`. The demo below demonstrates an example UX.
```

    IDV capabilities when it's available. They collect email and phone from the
    user (Richard) and check if the user has matching credentials. The user
    does, so Kredita lets them verify with 1-Click.
  </p>
  <iframe
    src='https://marvelapp.com/prototype/9cab6ai?emb=1&iosapp=false&frameless=false'
    width='453'
    height='864'
    allowTransparency='true'
    frameborder='0'
  ></iframe>
</details>

{' '}

<details>
  <summary>Demo Lendtable Accepting Credentials via Partner Dashboard</summary>
  <p>
    This demo shows Lendtable accepting their partner, Hooli, credentials in
    order to gain 1-Click IDV capabilities. In this case Hooli can issue the
    credential in an on-demand fashion when the user clicks the button to redeem
    the Lendtable offer in the Hooli dashboard. This would ensure the user has
    the necessary credentials, so Lendtable can verify with 1-Click.
  </p>
  <iframe
    src='https://marvelapp.com/prototype/1952a259?emb=1&iosapp=false&frameless=false'
    width='453'
    height='864'
    allowTransparency='true'
    frameborder='0'
  ></iframe>
</details>

## Implementation

> **You need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, you can accept their credentials with the user consent at the click of button for the user. This is what we call 1-Click IDV.

Reviewer: ```suggestion
**You need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, they can share all of those credentials with 1-click.
```

<details>
  <summary>What counts as a match?</summary>
  <p>
    When you call <code>/hasMatchingCredentials</code>, you include one or more
    credential requests. For each credential request, you specify:
    <ul>
      <li>the type</li>
      <li>(optional) which issuers are acceptable</li>
      <li>(optional) whether the credential is required</li>
    </ul>
  </p>
  <p>
    There's a match if, for each credential request for a required credential,
    the user has a credential of the correct type from an acceptable issuer.
  </p>
</details>
The user will either have matching credentials, or they won't. If they do, continue
to step (2). If they don't, you're done with Verified Inc. and should take the user

> through your existing verification flow.{' '}

Reviewer: ```suggestion
through your existing verification flow.
```
